index.html

```diff
@@ -73,10 +73,7 @@
         <div class="container" id="maincontent" tabindex="-1">
             <div class="row">
                 <div class="col-lg-12">
-                    <img class="img-responsive" src="img/profile.png" alt="code for poznan">
-                    <div class="intro-text">
-                        <h1 class="name">Code for Poznan</h1>
-                    </div>
+                    <img class="img-responsive cfp-logo" src="img/logo codeforpoznan.svg" alt="code for poznan">
```

Comment:
@agacix have you commited `img/logo codeforpoznan.svg` file? I don't see it in the pull request